src/features/bakery/components/CraftingItems.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useContext, useState } from "react";
+import { useActor } from "@xstate/react";
+import classNames from "classnames";
+
+import token from "assets/icons/token.png";
+
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import { Context } from "features/game/GameProvider";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { Craftable } from "features/game/types/craftables";
+import { InventoryItemName } from "features/game/types/game";
+
+interface Props {
+  items: Partial<Record<InventoryItemName, Craftable>>;
+  isBulk?: boolean;
+}
+
+export const CraftingItems: React.FC<Props> = ({ items, isBulk = false }) => {
```

Comment:
Is this file different from the other `craftingItems` file? I would suggest reusing the previous one for consistency instead of creating a new file